api-design-guidelines/data-modelling.md

```diff
@@ -0,0 +1,275 @@
+---
```

Comment:
This kind of metadata is only partially supported by GH (it is displayed as a regular table). We might want to remove it or at least get rid of irrelevant fields like "id", "image", etc.

Reply:
Yes, that was accidentally included. I hadn't saved the updates, thanks for pointing out :)